docs/guides/advanced/NavigatingOutsideOfComponents.md

```diff
@@ -20,7 +20,8 @@ And then import it to render a `<Router>`:
 ```js
 // index.js
 import history from './history'
-React.render(<Router history={history}/>, el)
+import { render } from 'react-dom'
```

Comment:
Sorry I mis-spoke - better to drop this import entirely (since it wasn't there before anyway).